build/get_python_test_deps.py

```diff
@@ -53,8 +60,27 @@ def _Untar(path):
   try:
     tar_file.extractall()
   finally:
+    dir_name = tar_file.getnames()
     tar_file.close()
-
+      # Return the first member in the returned list as it's the top dir
+      # name. Useful for passing on the top/first output dir name to other
+      # functions.
+    return dir_name[0]
+
+# Installs the requests python module.
+def _InstallRequests(requests_dir):
+  cwd = os.getcwd()
+  try:
+    print 'About to install requests module into your system python.'
+    os.chdir(requests_dir)
+    result = os.system('sudo python setup.py install')
```

Comment:
Yes there is but the reason I did it this way is because existing code already does it this way but will look into it.